CODE/Sources/PennyBOT/Bot.swift

```diff
@@ -20,26 +20,54 @@ struct Penny {
             try! client.syncShutdown()
             try! eventLoopGroup.syncShutdownGracefully()
         }
-
-        let options = SwiftcordOptions(isBot: true, willLog: true)
-        let bot = Swiftcord(token: ProcessInfo.processInfo.environment["BOT_TOKEN"] ?? "", options: options, logger: logger, eventLoopGroup: eventLoopGroup)
-
         
-        // Set activity
-        let activity = Activities(name: "Showing appreciation to the amazing Vapor community", type: .playing)
-        bot.editStatus(status: .online, activity: activity)
-
-        // Set intents
-        bot.setIntents(intents: .guildMessages)
-
-        bot.addListeners(MessageLogger(logger: logger, httpClient: client))
-        //let messageLogger = MessageLogger(bot: bot)
-        //messageLogger.messageLogger()
-
-        //let slashCommandListener = SlashCommandListener(bot: bot)
-        //slashCommandListener.BuildCommands()
-        //slashCommandListener.ListenToSlashCommands()
+        guard let token = ProcessInfo.processInfo.environment["BOT_TOKEN"],
+              let appId = ProcessInfo.processInfo.environment["BOT_APP_ID"] else {
```

Comment:
Yes, sorry I forgot to mention.
App id is not absolutely necessary but DiscordBM requires it as it'll make some api calls nicer. That could change, but for now this should not be a problem since app id is easily accessible. Its a public id which is also the user id of the bot account.